src/onboarding2/Create/Create.js

```diff
@@ -265,7 +399,12 @@ function Create() {
     >
       {status === STATUS_DEPLOYMENT ? (
         <Deployment
-          ready={signedTransactions === deployTransactions.length}
+          loadingTransactions={fetchingTemplateInformation}
```

Comment:
We may also want to show different UI in `DeploymentSteps` based on if we're still waiting for the ABI to be fetched before sending transactions.